tests/acceptance/test_files/chaos_and_query.bats

```diff
@@ -2,23 +2,39 @@ load "$LIB_BATS_ASSERT/load.bash"
 load "$LIB_BATS_SUPPORT/load.bash"
 
 @test "select * from chaos.chaos_high_row_count order by column_0" {
-  run steampipe query --output json  "select * from chaos.chaos_high_row_count order by column_0 limit 10"
-  assert_equal "$output" "$(cat $TEST_DATA_DIR/expected_1.json)"
+  steampipe query --output json  "select * from chaos.chaos_high_row_count order by column_0 limit 10" > output.json
+  run jd -f patch "$TEST_DATA_DIR/expected_1.json" output.json
+  diff=$($FILE_PATH/json_patch.sh $output)
```

Comment:
Similarly here, you don't need the diff from json_patch.sh script since we are not comparing snapshots.

All you need is:
```
@test "select * from chaos.chaos_high_row_count order by column_0" {
  steampipe query --output json  "select * from chaos.chaos_high_row_count order by column_0 limit 10" > output.json
  run jd "$TEST_DATA_DIR/expected_1.json" output.json
  echo $output
  assert_success
  rm -f output.json
}
```